index.html

```diff
@@ -3652,12 +3652,6 @@ <h3 id="applying">
         notation are defined in [[!ECMASCRIPT]].
       </p>
       <p>
-        When instructed to <dfn>Trim</dfn>(<var>x</var>), a user agent MUST
-        behave as if [[!ECMASCRIPT]]'s <a data-cite=
-        "!ECMASCRIPT#sec-string.prototype.trim">String.prototype.trim()</a>
```

Comment:
its seems that respec gets the list from http://www.specref.org/?q=ECMASCRIPT which shows https://tc39.github.io/ecma402/ (current editors draft) which is not what shows up in our spec